networkit/cpp/centrality/ApproxElectricalCloseness.cpp

```diff
@@ -507,6 +507,13 @@ void ApproxElectricalCloseness::run() {
             G.forNodes([&](node u) { rhs[u] = -1.0 / static_cast<double>(n); });
             rhs[root] += 1.;
             cg.solve(rhs, sol);
+
+            // ensure column sum of entries is 0.
+            double columnSum = 0.;
+            for (int i = 0; i < n; i++) {
+                columnSum += sol[i];
+            }
+            sol -= columnSum / n;
```

Comment:
There is an implicit `count` to `double` conversion here, use `columnSum / static_cast<double>(n)`